src/libraries/System.Private.CoreLib/src/System/Reflection/Emit/AssemblyBuilder.cs

```diff
@@ -32,8 +33,55 @@ public ModuleBuilder DefineDynamicModule(string name)
             return GetDynamicModuleCore(name);
         }
 
+        /// <summary>
+        /// Defines an <see cref="AssemblyBuilder"/> that can be saved to a file or stream.
+        /// </summary>
+        /// <param name="name">The name of the assembly.</param>
+        /// <param name="coreAssembly">The assembly that denotes the "system assembly" that houses the well-known types such as <see cref="object"/></param>
+        /// <param name="assemblyAttributes">A collection that contains the attributes of the assembly.</param>
+        /// <returns>An <see cref="AssemblyBuilder"/> that can be persisted.</returns>
+        /// <exception cref="ArgumentNullException">The <paramref name="name"/> or <paramref name="name.Name"/> or <paramref name="coreAssembly"/> is null.</exception>
+        /// <remarks>Currently the persisted assembly doesn't support running, need to save it and load back to run.</remarks>
+        public static AssemblyBuilder DefinePersistedAssembly(AssemblyName name, Assembly coreAssembly, IEnumerable<CustomAttributeBuilder>? assemblyAttributes = null)
+        {
+            ArgumentNullException.ThrowIfNull(name);
+            ArgumentNullException.ThrowIfNull(coreAssembly);
+
+            Type assemblyType = Type.GetType("System.Reflection.Emit.AssemblyBuilderImpl, System.Reflection.Emit", throwOnError: true)!;
+            ConstructorInfo con = assemblyType.GetConstructor(BindingFlags.NonPublic | BindingFlags.Instance, [typeof(AssemblyName), typeof(Assembly), typeof(IEnumerable<CustomAttributeBuilder>)])!;
+            return (AssemblyBuilder)con.Invoke([name, coreAssembly, assemblyAttributes]);
+        }
+
         protected abstract ModuleBuilder? GetDynamicModuleCore(string name);
 
+        /// <summary>
+        /// Serializes the assembly to stream.
```

Comment:
Suggestion: change "to stream" to:
"to a stream" or 
"to a <see cref="Stream"/>